keymaps/omnisharp-atom.cson

```diff
@@ -33,6 +33,7 @@
   'ctrl-alt-down': 'omnisharp-atom:navigate-down'
   'ctrl-.': 'omnisharp-atom:get-code-actions'
   'alt-enter': 'omnisharp-atom:get-code-actions'
+  'ctrl-shift-t': 'omnisharp-atom:find-symbols'
```

Comment:
This is already used by atom. Consult this: https://github.com/atom/atom/tree/master/keymaps :)